lib/super_good/engine.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module SuperGoodSolidusTaxjar
+  class Engine < Rails::Engine
+    isolate_namespace Spree
+    engine_name 'super_good-solidus_taxjar'
+
+    config.autoload_paths += %W(#{config.root}/lib)
```

Comment:
Is there a reason you're adding lib to autoloads? I don't think we should.

Reply:
No, there's no good reason. We have removed this.

We've also removed the rspec configuration, as it can be added when it is needed, in #37